src/views/Pages/Accout/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react'
+// import Dashboard from '../Dashboard'
+// import { isAuthEmail, authenticateUser, isAuthUserType } from '../../../services/Auth'
+import { Link } from 'react-router-dom';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Link' is defined but never used.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)